.github/workflows/release.yml

```diff
@@ -138,9 +134,21 @@ jobs:
         build-test-osx-x86,
         build-test-osx-arm64,
       ]
+    outputs:
+      manylinux-x86-wheel-artifact-name: ${{ steps.wait-for-build-test.outputs.manylinux-x86-wheel-artifact-name }}
```

Comment:
This solution looks even worse to me. But again why we introduce those intermediate step variables in the first place?